### DISCUSSION TITLE
problem on disconnecting

### DISCUSSION THREAD

**Discussion by theariansalmanzadeh** [rainbow-me/rainbowkit#1388] [Feedback]
there is a problem when using with trust wallet browser extension. after disconnecting the wallet is still connected and will not connect to any other wallet. I'd appreciate it if you fix this issue.

many thanks

**Comment by DanielSinclair** [comment]
This issue is resolved in RainbowKit [1.0.5](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%401.0.5) and beyond
